daemon/set.go

```diff
@@ -0,0 +1,58 @@
+package daemon
+
+import (
+	"fmt"
+
+	"github.com/docker/docker/runconfig"
+)
+
+// ContainerSet sets resources of the container
+func (daemon *Daemon) ContainerSet(name string, hostConfig *runconfig.HostConfig) ([]string, error) {
+	var warnings []string
+
+	warnings, err := daemon.verifyContainerSettings(hostConfig, nil)
+	if err != nil {
+		return warnings, err
+	}
+
+	if err := daemon.set(name, hostConfig); err != nil {
+		return warnings, err
+	}
+
+	return warnings, nil
+}
+
+func (daemon *Daemon) set(name string, hostConfig *runconfig.HostConfig) error {
+	if hostConfig == nil {
+		return nil
+	}
+
+	container, err := daemon.GetContainer(name)
+	if err != nil {
+		return err
+	}
+
+	if container.RemovalInProgress || container.Dead {
+		return fmt.Errorf("Container is marked for removal and cannot be set.")
```

Comment:
Very minor thing but can we wrap "set" in quotes?  W/o something, like quotes, it reads kind of funny - when someone first reads this they may ask "set to what?"  When really what we're trying to say is that the "set" function can not be used at this time.  Another option is to use the word "updated" but I think keeping it consistent with the action is probably better.